web-ui/src/main/resources/catalog/templates/admin/settings/sources.html

```diff
@@ -103,35 +103,62 @@
       <div class="panel-body">
         <div data-ng-show="source.type === 'subportal'">
           <form name="gnSourceForm">
-            <label data-translate="" class="gn-required">sourceUuid</label>
-            <input
-              type="text"
-              class="form-control"
-              data-ng-required="true"
-              data-ng-readonly="!isNew"
-              data-ng-disabled="::!user.isAdministrator()"
-              data-ng-pattern="/^[A-Za-z0-9\-_]+$/"
-              data-ng-model="source.uuid"
-            />
-            <p class="help-block" data-translate="">sourceUuid-help</p>
+            <div
+              data-ng-class="gnSourceForm.sourceId.$error.required
+                                || gnSourceForm.sourceId.$error.maxlength
+                                || gnSourceForm.sourceId.$error.pattern ? 'has-error' : ''"
+            >
+              <label class="control-label gn-required" data-translate=""
+                >sourceUuid</label
+              >
+              <input
+                type="text"
+                class="form-control"
+                name="sourceId"
+                data-ng-required="true"
+                data-ng-readonly="!isNew"
+                data-ng-disabled="::!user.isAdministrator()"
+                data-ng-pattern="/^[A-Za-z0-9\-_]+$/"
+                data-ng-maxlength="255"
+                data-ng-model="source.uuid"
+              />
+              <p class="help-block" data-translate="">sourceUuid-help</p>
+
+              <p
```

Comment:
If the identifier exceeds the maxlength, it is not displayed a message `Portal identifier can not exceed 255 characters!`, instead the help text `Unique identifier of the portal used in the URL. MUST be composed of letters, numbers, '-' and '_' only.` is highlighted in red. Seems a bit inconsistent.